### DISCUSSION TITLE
using spiffs to draw .bmp file onto a sprite

### DISCUSSION THREAD

**Discussion by griffinmt** [Bodmer/TFT_eSPI#2537] [General]
Cant get anything to work doing this. I have loaded the /data directory which contains my .bmp files into the esp32.
What do I need to include so I can draw one of the bmp files onto one of my sprites.

**Comment by Bodmer** [comment]
Look at the example:

https://github.com/Bodmer/TFT_eSPI/tree/master/examples/Generic/TFT_SPIFFS_BMP
